### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by amyerspigg** [Burning-Issues-Watersheds/5-bui-user.interface#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amyerspigg** [comment]
hello world! 🔥

**Comment by guerrero-fj** [comment]
Hello inded! Please introduce yourselves!

**Reply by guerrero-fj** [reply]
I'm not the world, FYI
